### PR TITLE
Add blog post on PostgreSQL state backend support for Pulumi

### PR DESCRIPTION
**Pull request by matanbaruch**
This pull request introduces a new blog post detailing the addition of PostgreSQL as a state backend for Pulumi's DIY backend, along with a new author profile for the contributor of this feature. The blog post outlines the technical implementation, community-driven development process, and future possibilities for the feature.

### Blog Post Addition:
* [`content/blog/postgres-diy-backend/index.md`](diffhunk://#diff-da6b560b494d84664d173c846dc78de4385419310dd1aaef22cc8def641d264fR1-R236): Added a comprehensive blog post titled "Adding PostgreSQL State Backend Support to Pulumi: A Community Contribution Journey." The post covers the motivation, technical challenges, implementation details, testing, and community impact of the new PostgreSQL state backend feature.

### Author Profile:
* [`data/team/team/matan-baruch.toml`](diffhunk://#diff-f43ab589d4da419ec44eb730c90df50ef7617277385d90af6c3f53e26bd5e354R1-R8): Added a new author profile for Matan Baruch, the community contributor behind the PostgreSQL backend feature. Includes social media links and contributor status.

### CODE REVIEW

File: content/blog/postgres-diy-backend/index.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+The feature was tested across Pulumi's existing CI/CD pipeline, including:
+- Multiple Go versions
+- Different operating systems
+- Various PostgreSQL versions
+- Performance benchmarking
+
+### Community Testing
+
+The PR remained open for several weeks, allowing community members to test the implementation and provide feedback. This collaborative approach helped identify edge cases and improve the overall quality.
+
```
Comment (adamgordonbell) [inline content/blog/postgres-diy-backend/index.md]: I think we could drop this section and focus this blog on those who would be interested users of this feature. Those curious about the implementation could see the PR.  
```suggestion

```

File: content/blog/postgres-diy-backend/index.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+**Advantages:**
+- Excellent performance for small to medium-sized state files
+- Strong consistency guarantees
+- Mature backup and recovery options
+- Comprehensive monitoring capabilities
+
+**Limitations:**
+- May be slower than object storage for very large state files
+- Requires database infrastructure management
+- Signed URLs (used for state permalinks) are not supported
```
Comment (adamgordonbell) [inline content/blog/postgres-diy-backend/index.md]: I'd like to get some feedback from PM/PMM at Pulumi on the comparison of this state backend to Pulumi cloud and make sure we properly highlight the trade offs. Let me get back to you.

**Review by adamgordonbell — CHANGES_REQUESTED** [review]
Overall this looks good. Thanks for the contribution both to Pulumi and to the blog!

I suggest removing the implementation part and I'm going to get some feedback on the limitations part internally.

**Comment by adamgordonbell** [comment]
@matanbaruch suggested rewording from product team at Pulumi here: https://github.com/matanbaruch/pulumi-docs/pull/1

**Comment by matanbaruch** [comment]
> @matanbaruch suggested rewording from product team at Pulumi here: https://github.com/matanbaruch/pulumi-docs/pull/1
> 
> 
> 
> 

Merged

**Review by adamgordonbell — APPROVED** [review]
